cfgov/jinja2/v1/job-description-page/index.html

```diff
@@ -120,12 +120,11 @@ <h4>Position type</h4>
             <a class="o-expandable_target o-expandable_link" title="Expand content">
                 <span class="o-expandable_cue o-expandable_cue-open">
                     Read full description
-                    <span class="cf-icon cf-icon-plus-round"></span>
+                    {{ svg_icon('plus-round') }}
                </span>
                 <span class="o-expandable_cue o-expandable_cue-close">
                     Hide full description
-                    <span class="cf-icon cf-icon-minus-round"></span>
-               </span>
+                    {{ svg_icon('plus-minus') }}
```

Comment:
Fix to `minus-round`

Reply:
@anselmbradford This will be removed when we back merge your PR.